easybuild/easyconfigs/t/Teneto/Teneto-0.5.1-foss-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,67 @@
+easyblock = 'PythonBundle'
+
+name = 'Teneto'
+version = '0.5.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://teneto.readthedocs.io'
+description = """Teneto is package for deriving, analysing and plotting temporal network representations.
+Additional tools for temporal network analysis with neuroimaging contexts."""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+
+dependencies = [
+    ('Python', '3.8.2'),
+    ('SciPy-bundle', '2020.03', versionsuffix),
+    ('matplotlib', '3.2.1', versionsuffix),
+    ('statsmodels', '0.11.1', versionsuffix),
+    ('networkx', '2.4', versionsuffix),
+    ('Nilearn', '0.7.0', versionsuffix),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    ('tqdm', '4.56.0', {
```

Comment:
@akesandgren any reason you included this as an extension rather than updating the standalone version?

Reply:
I didn't find a standalone version, might have forgotten to look for that one specifically. But if it exists i will use it...

Reply:
Fixed